newapi/ooniapi/pages/__init__.py

```diff
@@ -40,13 +40,25 @@ def index():
     return render_template("index.html")
 
 
+@pages_blueprint.route("/css/master.css")
+def serve_master_css() -> Response:
+    """Serve the Bootstrap CSS
+    ---
+    responses:
+      '200':
+        description: CSS
+    """
+    fn = "/usr/lib/python3/dist-packages/ooniapi/static/css/master.css"
```

Comment:
Why are we hardcoding the path to the ooniapi installation? Perhaps we can just serve it via `/static/css/` in the templates